### DISCUSSION TITLE
Pico W and Scoppy

### DISCUSSION THREAD

**Discussion by fhdm-dev** [fhdm-dev/scoppy#69] [General]
This is just a quick post in case anyone is wondering if the Scoppy firmware will work on the Pico W.

As I'm sure you know, the Pico W uses several of the RP2040 GPIOs for communicating with the wireless chip. The Scoppy firmware uses two of these, namely GPIO 25 for blinking the status LED and GPIO 23 for setting the power supply to low noise (PWM) mode.

The Pico W is not yet available where I live so I can't confirm the behaviour but my guess is that the firmware will work - except that the status LED will not blink and the signal traces you see in the app might appear more noisy.

A Pico W version of the firmware will be available once I can get my hands of one of these devices.

.... and I'm looking into the feasibility of adding wireless support to the app and firmware. Stay tuned :laughing:

**Comment by Abhijeetbyte** [comment]
> wireless support 

 That would be awesome

**Comment by fhdm-dev** [comment]
Done. The latest version of the app and firmware allows for wireless connection between the app and the firmware running on a Pico W.
See [Getting started with Scoppy and the Pico W](https://oscilloscope.fhdm.xyz/wiki/Getting-started-with-the-Pico-W) for more information.

This functionality is only available if you have upgraded to the premium version of the app.

**Comment by Kerner947** [comment]
Hi. Both the USB and W versions work perfectly for me. In the W version, CH2 was placed on pin A2 (34) and the generator was placed on pin GP16 (21). This is what I experienced. When viewed on a "real" oscilloscope, the signals of both are very nice. On the other hand, the rising edges on the raspberry oscilloscope placed on the test panel are quite ugly. I hope this will change when I build it in a final - shaded - form. Overall, I think it's a great, cheap device. Congratulation!.
